README.md

```diff
@@ -54,7 +54,15 @@ Clone the repository
 
   - Re-fetch the project: `$ fvm flutter pub get`
 
-- The project uses the package name, the app name and the project name of the template if `PACKAGE_NAME`, `APP_NAME` and `PROJECT_NAME` aren't specified.
+- The fields detail:
+
+  | Field name   | Is mandatory | Rationale                                                                       |
```

Comment:
I prefer to use these common words
```
Is mandatory -> Requires
Rationale -> Description
```
so it could make them easy to understand.

What do you think?